### PR TITLE
Fixed Auto Scroll Issue

### PR DESCRIPTION
**Pull request by sgadman**
This is a solution to fix the auto scroll issue described in Issue #59

### CODE REVIEW

**Comment by jacobworrel** [comment]
Thanks for submitting a PR @sgadman ! I'll review your changes when I have some time.

**Comment by dkertson-balto** [comment]
Bump. Would love to have these changes merged

**Comment by jacobworrel** [comment]
I have been swamped with work and havent been able to review these changes but am hoping to get around to it sometime. 😔

File: src/MenuList.tsx [hunk 1]
```diff
@@ -122,52 +124,71 @@ function MenuList (props) {
...
+              list.current.scrollToItem(currentIndex);
+            }
+          }
+        } else {
+          list.current.scrollToItem(currentIndex);
+        }
       }
     },
-    [currentIndex, children, list]
+    [currentIndex, children, list, ref]
```
Comment (vikkrantxx7) [inline src/MenuList.tsx:150]: ref is unnecessary in the dependency.

File: src/MenuList.tsx [hunk 2]
```diff
@@ -122,52 +124,71 @@ function MenuList (props) {
...
+
+      if (currentIndex >= 0 && list.current !== null && (refCurrent !== null && refCurrent !== undefined)) {
+        const currentItem = document.getElementById(children[currentIndex].props.innerProps.id)
+
+        if (currentItem) {
+          const currentItemRect = currentItem.getBoundingClientRect();
+          const menuRect = refCurrent.getBoundingClientRect();
+
+          if (Math.abs(currentItemRect.top - menuRect.bottom) <= Math.abs(currentItemRect.top - menuRect.top)) {
+            if (currentItemRect.top + 5 >= menuRect.bottom) {
```
Comment (vikkrantxx7) [inline src/MenuList.tsx:137]: why 5?

**Pull request closed without merging**